### PR TITLE
[DOCS] Adds size parameter details to aggregating data for faster performance page

### PR DESCRIPTION
**Pull request by szabosteve**
## Overview

This PR adds a recommendation for using the `size` parameter in a non-composite aggregation.

### Preview

[Aggregations]()

### CODE REVIEW

File: docs/reference/ml/anomaly-detection/ml-configuring-aggregations.asciidoc [hunk 1]
```diff
@@ -34,6 +34,10 @@ to match. For example, if you use a `max` aggregation on a time field called
 descending order. Additional `composite` aggregation value sources are allowed, 
 such as `terms`.
 
+* The `size` parameter of the non-composite aggregations must be configured
```
Comment (lcawl) [inline docs/reference/ml/anomaly-detection/ml-configuring-aggregations.asciidoc:37]: > must be configured properly to match

Could we just say "must match" instead?

**Review by lcawl — APPROVED** [review]
One comment, otherwise LGTM

**Comment by szabosteve** [comment]
@elasticsearchmachine test this please

**Comment by szabosteve** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.6|[](https://github.com/elastic/elasticsearch/pull/93480)|
|✅|8.5|[](https://github.com/elastic/elasticsearch/pull/93481)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**